unearthed-arcana/2022/20220523.xml

```diff
@@ -0,0 +1,961 @@
+			<set name="official">true</set>
+			<set name="playtest">true</set>
+			<set name="release">20220523</set>
+		</setters>
+	</element>
+
+	<!-- Barbarian: Path of the Giant -->
+	<element name="Path of the Giant" type="Archetype" source="Unearthed Arcana: Giant Options" id="ID_WOTC_UA20220523_ARCHETYPE_BARBARIAN_PATH_OF_THE_GIANT">
+		<supports>Primal Path</supports>
+		<description>
+			<p>Barbarians who walk the Path of the Giant draw strength from the primal forces that are the Giants and their elemental ilk. Their rages surge with elemental power and cause these barbarians to grow in size, transforming them into avatars of primordial might.</p>
+			<p class="indent">During their rage, a barbarian may take after a legendary storm giant, hair turning white and eyes crackling with lightning. Others adopt the monstrous features of apocalyptic titans, rending the world around them so that it may be made anew.</p>
+			<div element="ID_WOTC_UA20220523_ARCHETYPE_FEATURE_PATH_OF_THE_GIANT_GIANT_POWER" />
+			<div element="ID_WOTC_UA20220523_ARCHETYPE_FEATURE_PATH_OF_THE_GIANT_GIANTS_HAVOC" />
+			<div element="ID_WOTC_UA20220523_ARCHETYPE_FEATURE_PATH_OF_THE_GIANT_ELEMENTAL_CLEAVER" />
+			<div element="ID_WOTC_UA20220523_ARCHETYPE_FEATURE_PATH_OF_THE_GIANT_MIGHTY_IMPEL" />
+			<div element="ID_WOTC_UA20220523_ARCHETYPE_FEATURE_PATH_OF_THE_GIANT_DEMIURGIC_COLOSSUS" />
+		</description>
+		<sheet display="false" />
+		<rules>
+			<grant type="Archetype Feature" id="ID_WOTC_UA20220523_ARCHETYPE_FEATURE_PATH_OF_THE_GIANT_GIANT_POWER" level="3" />
+			<grant type="Archetype Feature" id="ID_WOTC_UA20220523_ARCHETYPE_FEATURE_PATH_OF_THE_GIANT_GIANTS_HAVOC" level="3" />
+			<grant type="Archetype Feature" id="ID_WOTC_UA20220523_ARCHETYPE_FEATURE_PATH_OF_THE_GIANT_ELEMENTAL_CLEAVER" level="6" />
+			<grant type="Archetype Feature" id="ID_WOTC_UA20220523_ARCHETYPE_FEATURE_PATH_OF_THE_GIANT_MIGHTY_IMPEL" level="10" />
+			<grant type="Archetype Feature" id="ID_WOTC_UA20220523_ARCHETYPE_FEATURE_PATH_OF_THE_GIANT_DEMIURGIC_COLOSSUS" level="14" />
+		</rules>
+	</element>
+
+	<element name="Giant Power" type="Archetype Feature" source="Unearthed Arcana: Giant Options" id="ID_WOTC_UA20220523_ARCHETYPE_FEATURE_PATH_OF_THE_GIANT_GIANT_POWER">
+		<requirements>!ID_INTERNAL_UA_FEATURE_REPLACEMENT_BARBARIAN_GIANT_GIANT_POWER</requirements>
+		<description>
+			<p><em>3rd-Level Path of the Giant Feature</em></p>
+			<p>When you choose this path, you learn to speak, read, and write Giant or one other language of your choice if you already know Giant. Additionally, you learn either the <i>druidcraft</i> or <i>thaumaturgy</i> cantrip (your choice). Wisdom is your spellcasting ability for this spell.</p>
+		</description>
+		<sheet>
+			<description>Wisdom is your spellcasting ability for the chosen cantrip.</description>
+		</sheet>
+		<rules>
+			<grant type="Language" id="ID_LANGUAGE_GIANT" />
```

Comment:
"you learn to speak, read, and write Giant or one other language of your choice if you already know Giant" 
From what i can see this line doesn't allow that to happen, it does the first bit fine but doesn't let you choose another if you already know giant.